templates/frontend/pages/catalogCategory.tpl

```diff
@@ -35,7 +35,7 @@
 	<div class="about_section{if $image} has_image{/if}{if $description} has_description{/if}">
 		{if $image}
 			<div class="cover" href="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="fullSize" type="category" id=$category->getId()}">
-				<img src="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="thumbnail" type="category" id=$category->getId()}" alt="{$category->getLocalizedTitle()|escape}" />
+				<img src="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="thumbnail" type="category" id=$category->getId()}" alt="{$category->getLocalizedTitle()|default:'null'}" />
```

Comment:
In this case, the alt tag only uses the category title. This is included in a `<h1>` tag in the breadcrumbs, so it doesn't provide any additional information about the image.

So for accessibility, in this case the `alt` attribute should _always_ be `null`.